org.civicrm.bootstrapcivihr/scss/common/placeholders.scss

```diff
@@ -0,0 +1,25 @@
+%search-filter-block {
```

Comment:
also create a `placeholders/` folder and put each placeholder in a different partial